content/getting-started/livechat-apps/webhook-apps/index.mdx

```diff
@@ -176,7 +178,7 @@ That would prevent situations in which the process takes longer than our webhook
 
 Note: LiveChat webhooks are sent with `Content-Type: application/json` header, so please make sure that your service can handle such requests.
 
-Note: `chat_changed` event type is sent only when the tag list for a particular chat was changed, no matter if the chat is pending or not.
+Note: `chat_changed` event type is sent only when the tag list for a particular chat was changed, no matter if the chat is currently ongoing or not.
```

Comment:
I wonder what the author originally meant by "pending"? 🤔 Queued, maybe?

Reply:
As far as I remember the chat_changed webhook triggers anytime a tag is changed, meaning during or after the chat - so the ongoing (in my mind) meant that the chat is currently live, not already archived :D